frontend/taipy-gui/src/components/Taipy/DateSelector.tsx

```diff
@@ -46,7 +56,9 @@ interface DateSelectorProps extends TaipyActiveProps, TaipyChangeProps {
     editable?: boolean;
     label?: string;
     width?: string | number;
-    analogic? :boolean;
+    analogic?: boolean;
+    disableDateConfig?: string;
+    defaultDisableDateConfig?: string;
```

Comment:
no need for this as it is not dynamic

Reply:
I would like to confirm the structure. Can I make these changes to the initial structure for the time?
> {
>     disableWeekdays?: boolean;
>     disableWeekends?: boolean;
>     disablePastDays?: boolean;
>     disableFutureDays?: boolean;
>     dayOfWeek?: number; // 0-6 (0 = Monday, ..., 6 = Sunday)
>     interval?: number;
>     oddInterval?:boolean;
>     occurence?: number;
> }

Or should I follow the new logic that is currently under discussion and has not been finalized after FabienLelaquais comment?

Reply:
we'll wait for @FabienLelaquais 's input